File: src/main/java/com/sixeyes/client/module/Category.java
package com.sixeyes.client.module;

public enum Category {
    COMBAT("Combat"),
    MOVEMENT("Movement"),
    PLAYER("Player"),
    RENDER("Render"),
    MISC("Misc");

    private final String name;

    Category(String name) {
        this.name = name;
    }

    public String getName() {
        return name;
    }
}

File: src/main/kotlin/com/sixeyes/client/extensions/Category.kt
package com.sixeyes.client.extensions

val COMBAT = Category("Combat")
val MOVEMENT = Category("Movement")
val RENDER = Category("Render")
val PLAYER = Category("Player")
val OTHER = Category("Other")

val categories = listOf(COMBAT, MOVEMENT, RENDER, PLAYER, OTHER)

open class Category(
    val name: String
)